build/int.cloudbuild.yaml

```diff
@@ -167,6 +167,26 @@ steps:
     - verify-quota-project-example
   name: 'gcr.io/cloud-foundation-cicd/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS'
   args: ['/bin/bash', '-c', 'cft test run TestQuotaProject --stage destroy --verbose']
+- id: init-tags-project-example
+  waitFor:
+    - create-all
+  name: 'gcr.io/cloud-foundation-cicd/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS'
+  args: ['/bin/bash', '-c', 'cft test run TestTagsProject --stage init --verbose']
```

Comment:
This should be safe with bpt v0.12+, however do you still want to init all at the start?

Reply:
I removed the single init for now